README.md

```diff
@@ -1,13 +1,22 @@
-# Project Name
+# Chatbot
 
 Replace this readme with your own information about your project.
 
 Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
 ## The problem
-
 Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
 
-## View it live
+We had problems with understanding how javascript worked at first. We had a tough time even starting out the code, and it took us along time to actually understand what we did and how to write in javascript.
+
+We approached our problems with an open mind and we asked for help from our group. They came in clutch, they are great! We wanted to fix our problems on our own, as we both felt that we could learn a lot by just trying out stuff. But when we didn't get enywhere, we asked for help and it was very useful. 
+
+We did not have much of any plan going into the project as we just follwed to different sections that were setup for us from the technigo brief.
 
+If we had more time we would added more questions to the bot.
+
+We also had a rough start as me (Harry) was ill and did not really have the strength to sit and code for long. I really feel that Maria was very understanding of this and want to thank Maria for this as I felt I dragged her down alittle bit for not being able to put 100% into the project from the start. 
```

Comment:
We all have bad days/weeks when we struggle, the important thing is that you didn't give up. Remember that you did your best under the circumstances.